src/pages/workspace/expensifyCard/issueNew/ConfirmationStep.tsx

```diff
@@ -144,9 +142,9 @@ function ConfirmationStep({policyID, backTo}: ConfirmationStepProps) {
                 <ValidateCodeActionModal
                     handleSubmitForm={submit}
                     isLoading={issueNewCard?.isLoading}
-                    sendValidateCode={() => requestValidateCodeAction()}
+                    sendValidateCode={requestValidateCodeAction}
+                    validateCodeActionErrorField="createAdminIssuedVirtualCard"
```

Comment:
`createAdminIssuedVirtualCard` is only returned for virtual cards and this screen handles both physical and virtual cards. We should have checked the card type and use the correct error field accordingly:
- virtual: createAdminIssuedVirtualCard
- physical: createExpensifyCard

Coming from https://github.com/Expensify/App/issues/61609